### ISSUE TITLE
REACT Visual: Can Instrument view text be reduced

### ISSUE THREAD

**Issue by HayleyMills** [CLOSER-Cohorts/archivist#613]
The response domain details can be made shorter:
-  Code list cardinality to be changed from "Min Responses : 1 Max Responses : 1" to "Min: 1 Max: 1"
![image](https://user-images.githubusercontent.com/16664703/141119736-c2820e1e-9f12-47a4-a85f-c96f9c480d4f.png)

- Text responses length to be changed from "(255 maximum length)" to "(max: 255)"
![image](https://user-images.githubusercontent.com/16664703/141119761-b8509603-5546-4d36-9ed9-4c7944a83dc4.png)

- Date response 'Date' not needed. 
![image](https://user-images.githubusercontent.com/16664703/141119995-e55c2e0c-9956-4812-9f03-435bb7e57e28.png)

**Comment by simonreed** [comment]
@beckyoldroyd this is on staging now

**Comment by beckyoldroyd** [comment]
@simonreed the response domains look as they should, but it looks like the entire box has been greyed out, rather than just the response domain and interviewee? E.g.:

<img width="734" alt="image" src="https://github.com/user-attachments/assets/b06cc026-ebc3-4aca-91ec-e4788a1db296">

**Comment by simonreed** [comment]
@beckyoldroyd sorry yes that was intentional along with https://github.com/CLOSER-Cohorts/archivist/issues/600 it appeared that the underlying issue was a lack of visual hierarchy so I've made it cleared what the main question text is and all background fields are in the divider blow in smaller fond. The box is the question item and question grid is to allow for the grey text to be easier to see.

**Comment by beckyoldroyd** [comment]
@simonreed ah ok, thanks for clarify. I'd like to ask the team what they think (since they're the main users) before we close the issue if that's ok?

**Comment by simonreed** [comment]
@beckyoldroyd of course, no problem. I'd also like in this phase of work (but in another issue) to make the statements and conditions on this page also look more in keeping with what we decide to do with the questions.

**Comment by beckyoldroyd** [comment]
@simonreed the team like the shortened response domains, but would prefer to keep the text black as it's difficult to read in grey, especially on a white background. So could we keep the shortened response domains, but go back to the blue background and black text please?

**Comment by beckyoldroyd** [comment]
@simonreed we've had further feedback about the position of the Interviewee field - having it before the instruction and response domain makes it harder to scan through the questionnaire to check the question instructions and responses. So please can we keep the order the same?

**Comment by beckyoldroyd** [comment]
@simonreed feedback from CA after testing on staging: please can we reduce the size of the question text, as it takes up a lot of space which can be distracting, especially for longer question texts (see example below). Please can we also make the greyed-out text a bit darker, as it's difficult to read when it's as light as it currently is. Thank you.

<img width="674" alt="image" src="https://github.com/user-attachments/assets/7274e7c5-0efc-48f5-b982-92056871f964">

**Comment by simonreed** [comment]
@beckyoldroyd do we know which page that screenshot is from? Then I can show you what an alternative might look like.

**Comment by beckyoldroyd** [comment]
@simonreed yes sure: https://closer-archivist-staging.herokuapp.com/instruments/main06. Thank you.

**Comment by simonreed** [comment]
@beckyoldroyd  is this ok? <img width="1128" alt="Screenshot 2024-12-13 at 11 11 35" src="https://github.com/user-attachments/assets/2aa03b73-07f6-4fcf-9988-a9fd0fd62905" />

**Comment by beckyoldroyd** [comment]
@simonreed the question text looks good. Is it possible to move the interviewee in the original position (i.e. before the question text)? I also think we should keep all the text black - shortening the response domains helps enough without needing to grey it out, is the feedback I've received from the team. Thanks!

**Comment by simonreed** [comment]
@beckyoldroyd  option A : <img width="1126" alt="Screenshot 2024-12-13 at 14 43 44" src="https://github.com/user-attachments/assets/669c7e84-0929-4363-b67e-1d2f3201204e" />

or option B : 
<img width="1108" alt="Screenshot 2024-12-13 at 14 45 29" src="https://github.com/user-attachments/assets/abc9e953-503d-402c-8e01-999db61cd806" />

**Comment by beckyoldroyd** [comment]
@simonreed option B please :) 

Please could we also remove the line running underneath the question text? The bold headings are helpful enough in separating the different parts of the question. Thank you!

**Comment by simonreed** [comment]
@beckyoldroyd updated on staging

**Comment by beckyoldroyd** [comment]
@simonreed the structure of the page looks good, but it looks like the text is still grey?

**Comment by beckyoldroyd** [comment]
Text changed to back and has been deployed to all Archivist instances
